### PR TITLE
[Rule Tuning] Svchost spawning Cmd

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Exclude noisy false positives related to common system activity.

### CODE REVIEW

File: rules/windows/execution_command_shell_started_by_svchost.toml [hunk 1]
```diff
@@ -19,9 +19,15 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where event.type in ("start", "process_started") and
+process where event.type == "start" and
   process.parent.name : "svchost.exe" and process.name : "cmd.exe" and 
-  not (process.pe.original_file_name == "Cmd.Exe" and process.args : "?:\\Program Files\\Npcap\\CheckStatus.bat??")
+  not (process.pe.original_file_name : "cmd.exe" and process.args : (
+    "??:\\Program Files\\Npcap\\CheckStatus.bat?",
```
Comment (w0rk3r) [inline rules/windows/execution_command_shell_started_by_svchost.toml:25]: Uses ? to match double quotes, to avoid the use of wildcards

Comment (terrancedejesus) [inline rules/windows/execution_command_shell_started_by_svchost.toml:25]: Question - What does "??" allow for. I understand "?" is a wildcard but doubled I am not familiar with.

**Review by terrancedejesus — APPROVED** [review]
LGTM. Minor question for me to learn.

File: rules/windows/execution_command_shell_started_by_svchost.toml (hunk 1 above)
Comment (w0rk3r) [reply rules/windows/execution_command_shell_started_by_svchost.toml:25]: The second one covers the disk letter

**Review by brokensound77 — APPROVED** [review]

**Review by Mikaayenson — APPROVED** [review]
LGTM

**Pull request merged**